dxcherry/src/test/runTest.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
```

Comment:
Why do u use 4 spaces here and 2 spaces another places? Let's make it consistent